### DISCUSSION TITLE
Examples of all the animations

### DISCUSSION THREAD

**Discussion by shoshinobi** [AntoineW/luge#22] [General]
This is a great find for a motion designer heading into dev. It would be awesome to have visual examples of the animations to make it easier to choose and learn the process. Thanks!

**Comment by AntoineW** [comment]
Hey, thank you 🙏

You're definitely right, I have to provide more examples! I started to add a new one here for scroll reveal animations, and I'll try to add more regularly.

👉 https://codepen.io/tag/luge

**Reply by shoshinobi** [reply]
Amazing! That helps so much. Appreciate you putting those up!

I've been eyeing the text echo reveals and the box that comes on white then wipes to reveal the content......magic. 
I'm guessing now it's with "Layer to right"? Are you revealing two separate divs to get that sweet echo effect?

**Reply by AntoineW** [reply]
The `layer-to-...` animations use `:after` pseudo element for the top layer and a _clip-path_ transition for the reveal, you don't need two divs.

**Reply by shoshinobi** [reply]
Slick. That's a great feature to showcase in the examples or in a future tutorial.

**Comment by fgarcia93mdz** [comment]
hello, what you have done is incredible, I would like to know a little more about how to use the library to make a Sticky elements
